src/python/pants/backend/python/util_rules/interpreter_constraints.py

```diff
@@ -93,6 +93,10 @@ def __str__(self) -> str:
     def debug_hint(self) -> str:
         return str(self)
 
+    @property
+    def description(self) -> str:
+        return str(sorted(str(c) for c in self))
```

Comment:
(same. and the implementation was stolen from `flake8`'s rules, as well as `bandit`'s)